aqua/qt5/Portfile

```diff
@@ -19,6 +19,12 @@ version             ${qt_version}
 set middle_name     everywhere-opensource
 set worksrcdir_middle_name everywhere
 
+# qt-base is broken on macOS 15+, but that locks us in to this on all of them for now,
```

Comment:
no idea what the second part of this comment is supposed to mean...

Reply:
I'll rephrase.